cmd/vela-worker/worker.go

```diff
@@ -41,6 +41,7 @@ type (
 	Config struct {
 		API      *API
 		Build    *Build
+		CheckIn  *time.Duration
```

Comment:
Wonder if we should consider not making this a pointer?

Similar to the `Hostname` field, then you wouldn't have to dereference this anytime you use the value 👍